sematic/db/queries.py

```diff
@@ -174,6 +176,38 @@ def get_run_status_details(
     return result_dict
 
 
+@dataclass
+class PipelineMetrics:
```

Comment:
Same comment for reserving official-sounding names. Why not use `BasicPipelineMetrics`, to mirror the Dashboard code?